views/partials/base.tmpl

```diff
@@ -53,6 +53,7 @@
 {{block "footer-block" .}}
     {{template "footer" .}}
 {{end}}
+<script>var MyRadioAPIKey = {{.PageContext.MyRadioAPIKey }};</script>
```

Comment:
Not sure this should be a thing. i believe at present the api key is not exposed anywhere? (the config in this repo excepted) Maybe there should be a "public apikey" config variable? That said of course, everything on the site is "public", so maybe it doesn't matter

Reply:
I believe this same notation is used elsewhere